src/components/Fund/FundBudgets/FundBudgetsContainer.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { stripesConnect } from '@folio/stripes/core';
+import {
+  Accordion,
+  Button,
+  Icon,
+} from '@folio/stripes/components';
+
+import { budgetsResource } from '../../../common/resources';
+import FundBudgets from './FundBudgets';
+
+const FundBudgetsContainer = ({
+  budgetStatus,
+  currency,
+  fundId,
+  hasNewBudgetButton,
+  history,
+  labelId,
```

Comment:
I wonder if we really need this in all components. They doesn't look like common components to me, I would expect they are used once without changing label